src/logic/safe/store/actions/transactions/fetchTransactions/index.ts

```diff
@@ -14,43 +13,43 @@ import { addOrUpdateCancellationTransactions } from 'src/logic/safe/store/action
 import { addOrUpdateTransactions } from 'src/logic/safe/store/actions/transactions/addOrUpdateTransactions'
 import { AppReduxState } from 'src/store'
 
-const noFunc = () => {}
```

Comment:
Type function return?

Reply:
it's deleted

Reply:
my bad, generated bad the comment, will create a new one